templates/localpathprovisioner.yaml

```diff
@@ -12,6 +12,9 @@ metadata:
 spec:
   kubernetes:
     minSupportedVersion: v1.14.0
+  cloudProvider:
+    - name: none
+      enabled: true
```

Comment:
Note: The default should be localvolumeprovisioner for none, but I believe we want to let people choose the localpath as an option (although not enabled by default).

Reply:
These is always confusing me, so if this property means that it'll appear enabled then we don't want it because it will create a conflict with the localvolumeprovisioner.

Reply:
I dont mind making that change, as long as we have concensus?

Reply:
I shared what it was discussed in the storage meetings 🤔 , localvolumeprovisioner should be the default for none unless the user specifies the opposite.

Reply:
Conclusion: Please, set the value to false